tasks/vmaas_sync/package_sync.go

```diff
@@ -17,6 +17,8 @@ import (
 
 const chunkSize = 10 * 1024
 
+type pkgSlice []models.Package
```

Comment:
- There is a rule to not abbreviate in larger scopes.
- We should avoid names like `pkgSlice`, `orgIDsMap`, `nameString` or `IDInt64`, whatever. The information about it being slice is there already.

Pick one, or suggest something else.
```suggestion
type pkgs []models.Package
```
or
```suggestion
type packages []models.Package
```

Reply:
I found out we have `models.PackageSlice` already, so I used that. We can refactor all these names at once in a separate PR. I think we have more important tasks right now.